can-view-parser.js

```diff
@@ -435,7 +445,7 @@ HTMLParser.parseAttrs = function(rest, handler){
 };
 
 HTMLParser.searchStartTag = function (html) {
-	var closingIndex = html.indexOf('>');
+	var closingIndex = HTMLParser.sanitizeAttrs(html).indexOf('>');
```

Comment:
this is a nifty solution ... is HTML everything that's left in the template?  If this is, this might be creating an `n^2` situation as this gets called on all HTML remaining for every attribute.